zipkin-ui/templates/trace.mustache

```diff
@@ -4,13 +4,14 @@
     <li class=''><a href='#'><strong>Services:</strong> <span class='badge'>{{services}}</span></a></li>
     <li class=''><a href='#'><strong>Depth:</strong> <span class='badge'>{{depth}}</span></a></li>
     <li class=''><a href='#'><strong>Total Spans:</strong> <span class='badge'>{{totalSpans}}</span></a></li>
+    <li class='navbar-right'><a href='/api/v1/trace/{{ traceId }}' id='traceJsonLink'><button type="button" class="btn btn-primary btn-xs badge" value="uiShowJson">JSON</button></a></li>
```

Comment:
No, it doesn't have the value property. How about using an explicit click handler on the a element? I think that's cleaner overall.